src/custom/state/sentry/updater/index.ts

```diff
@@ -3,44 +3,71 @@ import * as Sentry from '@sentry/browser'
 
 import useIsWindowVisible from 'hooks/useIsWindowVisible'
 import { useSwapState } from 'state/swap/hooks'
-import { SupportedChainId } from 'constants/chains'
 import { useCurrency } from 'hooks/Tokens'
 import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useAppSelector } from 'state/hooks'
+
+function _getSentryChainId(appChainId: null | number, chainId: number | undefined) {
+  const connectedChainId = chainId
+  const appChainIdFull = appChainId ?? undefined
```

Comment:
I don't quite get this variable name 🤔

Reply:
haha yeah me neither, changes were made and I never fully commited, will revisit